add page for microsoft build event (#2816)
pulumi/pulumi-hugo: themes/default/content/microsoft-build/_index.md

---
title: Microsoft Build <br/> May 23–25, 2023
meta_desc: Join us to find out how Pulumi’s Infrastructure as Code reduces cloud complexity, accelerates development velocity and helps companies scale and innovate.
meta_image: /images/microsoft-build/meta.png
type: page
layout: microsoft-build

links:
    items:

>         - heading: RSVP to Seattle Meetup

Reviewer: ```suggestion
        - heading: Join Us for Happy Hour
```

>           description: Join us at our re-launch of the Seattle Pulumi User Group to meet our CEO, Joe Duffy, and members of our product and engineering teams. 5PM on May 23rd at Rhein Haus Seattle.

Reviewer: ```suggestion
          description: No speakers, no agenda. Just a casual evening to network and have fun! Meet fellow engineers and CEO Joe Duffy and CTO Luke Lohan. Food and drinks are on us. May 23, 5:00 Pm PDT at Rhein Haus Seattle.
```

          action: Reserve your spot

>           link: https://www.meetup.com/pulumi-seattle/events/293452944/

Reviewer: ```suggestion
          link: https://info.pulumi.com/en/pug-meetup/happy-hour
```
